src/main/java/Duke.java

```diff
@@ -1,10 +1,78 @@
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents main class for chat bot.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    public static void main(String[] args) throws IOException {
+        Ui ui = new Ui();
+        ui.printIntro();
+        Scanner scan = new Scanner(System.in);
+
+        Storage storage = new Storage(System.getProperty("user.dir") + "/data/duke.txt");
+        Parser parser = new Parser();
+        TaskList taskList = new TaskList(storage, ui);
+
+        try {
+            storage.loadFileContents(taskList.tasks);
+        } catch (FileNotFoundException e) {
+            storage.createFile();
+        }
+
+        loop:
+        while (scan.hasNextLine()) {
+
+            ui.printLine();
+
+            ArrayList<String> result = parser.parseInputToList(scan);
+            String command = result.get(0);
+
+            String description, date;
+            int taskIndex;
+
+            switch (command) {
```

Comment:
Hi, I think for this the case: statements should be aligned with the switch statement instead of a 4 spaces indentation.

Reply:
Hi, thanks for letting me know! I must have missed it when I was adjusting everything.